config.yaml

```diff
@@ -2,38 +2,21 @@ debug: true
 db_log: postgres://exchange:exchange_AA9944@127.0.0.1/exchange 
 db_history: postgres://exchange:exchange_AA9944@127.0.0.1/exchange 
 assets:
-  - name: BCH
-    prec_save: 20
-    prec_show: 8
-  - name: BTC
-    prec_save: 12
-    prec_show: 8
-  - name: LTC
-    prec_save: 12
-    prec_show: 8
   - name: ETH
-    prec_save: 12
-    prec_show: 8
-  - name: ETC
-    prec_save: 12
-    prec_show: 8
-  - name: ZEC
-    prec_save: 12
-    prec_show: 8
-  - name: XMR
-    prec_save: 12
-    prec_show: 8
-  - name: DSH
-    prec_save: 12
-    prec_show: 8
+    prec_save: 6
+    prec_show: 6
+  - name: USDT
+    prec_save: 6
+    prec_show: 6
 markets:
-  - name: ETH_BTC
+  - name: ETH_USDT
     base:
       name: ETH
-      prec: 8
+      prec: 4
```

Comment:
Binance uses 5?

![image](https://user-images.githubusercontent.com/37070449/104864955-8d346900-5975-11eb-8a08-eb2e9766f628.png)

Reply:
It is only for test now.  2/4/6 seems more friendly to my eyes compared to 3/5/7.